snapper.py

```diff
@@ -33,6 +33,8 @@
 
 validate(instance=config, schema=schema)
 
+scrub_count_file = get_relative_path(__file__, './snapper.scrubCount')
```

Comment:
I'm a bit conflated on the naming, because we are counting sync counts, not scrub counts, correct? After N syncs, we perform a scrub. I would rather name the file and subsequent variables `sync_count`.